distribution/docs/src/main/resources/content/_configuring/connecting-to-sources.adoc

```diff
@@ -1,10 +1,30 @@
 :title: Connecting to Sources
 :type: configuration
-:status: draft
+:status: published
 :parent: Configuring Federation
-:order: 01
+:order: 04
 :summary: Connecting to sources.
-////
+
 == {title}
-TODO: https://codice.atlassian.net/browse/DDF-3349 - Add general administrator-level directions for connecting to sources
-////
+
+A *source* is a system consisting of a catalog containing Metacards.
+
+Catalog sources are used to connect Catalog components to data sources, local and remote.
```

Comment:
❓ `${ddf-catalog}` here? Are there rules for when to use or don't use it?

Reply:
The property is to reference the Catalog application itself. This distinction may not be as useful as it once was, but that's the genesis of it.